fetchai/ledger/api/__init__.py

```diff
@@ -53,10 +51,10 @@ def sync(self, txs: Transactions):
         else:
             raise TypeError('Unknown argument type')
 
-        limit = timedelta(minutes=2)
+        limit = timedelta(seconds=30)
```

Comment:
I would like to keep the default reasonably long for the time period. Can you maybe more this to a defaulted parameter i.e:

```python
def sync(self, txs: Transactions, timeout=None):
    timeout = int(timeout or 120)

    ...

    limit = timedelta(seconds=timeout)
```

Reply:
Done.